base/java-tools/src/com/netscape/cmstools/kra/KRACertTransportExportCLI.java

```diff
@@ -0,0 +1,120 @@
+// --- BEGIN COPYRIGHT BLOCK ---
+// This program is free software; you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation; version 2 of the License.
+//
+// This program is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU General Public License for more details.
+//
+// You should have received a copy of the GNU General Public License along
+// with this program; if not, write to the Free Software Foundation, Inc.,
+// 51 Franklin Street, Fifth Floor, Boston, MA 02110-1301 USA.
+//
+// (C) 2019 Red Hat, Inc.
+// All rights reserved.
+// --- END COPYRIGHT BLOCK ---
+
+package com.netscape.cmstools.kra;
+
+import java.io.FileOutputStream;
+import java.io.PrintStream;
+
+import org.apache.commons.cli.CommandLine;
+import org.apache.commons.cli.Option;
+import org.dogtagpki.cli.CLI;
+import org.dogtagpki.util.logging.PKILogger;
+import org.mozilla.jss.netscape.security.util.Cert;
+
+import com.netscape.certsrv.cert.CertData;
+import com.netscape.certsrv.client.PKIClient;
+import com.netscape.certsrv.system.SystemCertClient;
+
+/**
+ * @author Endi S. Dewata
+ */
+public class KRACertTransportExportCLI extends CLI {
+
+    public KRACertCLI certCLI;
+
+    public KRACertTransportExportCLI(KRACertCLI certCLI) {
+        super("transport-export", "Export transport certificate", certCLI);
+        this.certCLI = certCLI;
+
+        createOptions();
+    }
+
+    public void printHelp() {
+        formatter.printHelp(getFullName() + " [OPTIONS...]", options);
+    }
+
+    public void createOptions() {
+        Option option = new Option(null, "output-format", true, "Output format: PEM (default), DER");
+        option.setArgName("format");
+        options.addOption(option);
+
+        option = new Option(null, "output-file", true, "Output file");
+        option.setArgName("file");
+        options.addOption(option);
+
+        options.addOption("v", "verbose", false, "Run in verbose mode.");
+        options.addOption(null, "debug", false, "Run in debug mode.");
+        options.addOption(null, "help", false, "Show help message.");
```

Comment:
This is not required because this will be overwritten by `CLI`'s option https://github.com/dogtagpki/pki/blob/master/base/common/src/org/dogtagpki/cli/CLI.java#L343